src/InstrumentationEngine/ProfilerManager.h

```diff
@@ -241,11 +238,8 @@ namespace MicrosoftInstrumentationEngine
             _In_  LPVOID lpParameter
             );
 
-
         HRESULT LoadInstrumentationMethods(_In_ BSTR bstrConfigPath);
 
-        HRESULT ApplyInstrumentationMethodInstrumentation(_In_ CMethodInfo* pMethodInfo);
```

Comment:
I assume this didn't have an implementation.

Reply:
That's correct, there were two methods that did not have implementation so I've removed them.